targets/TARGET_NUVOTON/TARGET_M251/serial_api.c

```diff
@@ -448,6 +448,11 @@ int serial_writable(serial_t *obj)
     return ! UART_IS_TX_FULL(((UART_T *) NU_MODBASE(obj->serial.uart)));
 }
 
+int serial_tx_empty(serial_t *obj)
+{
+    return UART_GET_TX_EMPTY(((UART_T *) NU_MODBASE(obj->serial.uart)));
+}
+
```

Comment:
`UART_GET_TX_EMPTY(...)` indicates if TX FIFO is empty, but `UART_IS_TX_EMPTY(...)`, more complete, indicates if the last byte has transmitted from shift register. So please substitute `UART_IS_TX_EMPTY(...)` for `UART_GET_TX_EMPTY(...)` on all Nuvoton targets.

Reply:
Thanks @ccli8. I've fixed all Nuvoton targets with commits  c3936b2766ba31d96250d1b3375e4c496e1baf54 37d8846c2920fd347aa5d4e32fb08c06172f6b6d a7b15a5ba62675c850d95afbf4b4c0ecee4dc3dc 4748e2d48546a84519dd6d73d3ec05d75858e189 0454814bb054f6d5b45ac76ab44b45eb64522568 ae331a4830276a73a25387c023e6214d02060eb7.

Reply:
@pennam Thanks for your modificaiton